INSTRUCTION.md

```diff
@@ -0,0 +1,10 @@
+In order to set up the secrets and the config map:
+
+kubectl apply -f .infrastructure/configMap.yml
+kubectl apply -f .infrastructure/secret.yml
+
+In order to validate that the environment variables are set correctly:
+
+kubectl get pods
+kubectl exec -it <pod-name> -- /bin/sh
+prinenv
```

Comment:
There is a typo in the command `prinenv`. It should be `printenv` to correctly display the environment variables inside the pod.